javascript/javascript1/week3/Songs/song.js

```diff
@@ -19,6 +19,15 @@ const songDatabase = [{
   artist: 'The spies girls',
 },
 ];
+const song1 = new Object();
```

Comment:
or you can write: 
`const song1 = {
   songId: 5,
   title: "Summer of 69",
   artist:"Bryan Adams"
}`